errors/no-stylesheets-in-head-component.md

```diff
@@ -1,16 +1,16 @@
 # No Stylesheets In Head Component
 
-### Why This Error Occurred
+### Why This Error Occurred?
```

Comment:
We use the same format for all error files, which do not have the question mark or colon on the other heading. Can we be consistent?

Reply:
Yup that's fine.